service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsOrderDatesHandler.java

```diff
@@ -36,21 +36,19 @@
 
     @Async
     @EventListener
-    public void populateSDODates(PopulateSDODatesEvent event) {
+    public void populateStandardDirectionsDates(PopulateStandardDirectionsOrderDatesEvent event) {
```

Comment:
I would probably just call it `populateDates` seeing as we know from the context of the class what we are populating it for

Reply:
agree, changed